web/src/input.rs

```diff
@@ -1,125 +1,159 @@
-use ruffle_core::events::{KeyCode, TextControlCode};
+use ruffle_core::events::{
+    KeyDescriptor, KeyLocation, LogicalKey, NamedKey, PhysicalKey, TextControlCode,
+};
+use web_sys::KeyboardEvent;
 
-/// Convert a web `KeyboardEvent.code` value into a Ruffle `KeyCode`.
```

Comment:
This comment seems at least somewhat useful with s/KeyCode/PhysicalKey/.

Reply:
Added comments